openapi/p4pa-migration.openapi.yaml

```
@@ -6,7 +6,7 @@ info:
 servers:
   - url: http://localhost:8080/migration
 paths:
-  /{migrationFileType}/{organizationId}:
+  /{migrationFileType}/{orgIpaCode}:
```

let's use a shape similar to pu-sil
```suggestion
  /organization/{orgIpaCode}/{migrationFileType}:
```